civitai-downloader/civitai_downloader/utils.py

```
@@ -0,0 +1,185 @@
+    }
+    
+    multiplier = units.get(unit, 1)
+    return int(value * multiplier)
+
+
+def get_terminal_size() -> Tuple[int, int]:
+    """Get terminal size (columns, rows)."""
+    try:
+        size = os.get_terminal_size()
+        return size.columns, size.lines
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)